src/ixdat/techniques/ms.py

```diff
@@ -1189,12 +1194,16 @@ def gas_flux_calibration_curve(
             )
         else:
             cal_type = "gas_flux_calibration_curve"
+            mol_conc_ppm = 10**6
+            carrier_mol = mol
         for tspan, mol_conc_ppm, pressure in zip(tspan_list, mol_conc_ppm_list, p_list):
             t, S = measurement.grab_signal(mass, tspan=tspan, tspan_bg=tspan_bg)
             if axes_measurement:
-                axes_measurement.plot(t, S, color=STANDARD_COLORS[mass], linewidth=5)
-                mol_conc_ppm = 10**6
-                carrier_mol = mol
+                if axes_measurement_raw == True:
```

Comment:
Don't use `==` when comparing to `True`. Use `is`, or if other stuff that evaluates as a boolean to `True` is okay, just use the thing itself. So here, `if axes_measurement_raw: `
flake8 enforces this.

Reply:
fixed.